src/utilities/__tests__/extendSchema-test.js

```diff
@@ -884,11 +980,11 @@ describe('extendSchema', () => {
   });
 
   it('does not allow replacing a default directive', () => {
-    const ast = parse(`
+    const sdl = `
```

Comment:
I like this, but want to make sure I understand our guidelines for `sdl` vs `ast`: `sdl` => used when we have `TypeSystemDefinitionNode` types, and `ast` for `ExecutableDefinitionNode` types?

Reply:
@mjmahone SDL - is a string containing `TypeSystemDefinitions`
AST - a result of parse being called on any string in GraphQL syntax.